build.gradle

```diff
@@ -7,13 +7,13 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.0.0'
+        classpath 'com.android.tools.build:gradle:1.1.0'
```

Comment:
Unfortunately, no- I've been upgrading it every time that Android Studio tells me to, rolling to the latest version. The good news is that this shouldn't matter to customers (they don't have to upgrade their tools to use the library)